### DISCUSSION TITLE
Increase Rudder throw in acro

### DISCUSSION THREAD

**Discussion by NiteRavenX** [iNavFlight/inav#8238] [Q&A] [answered]
I am setting up a sport plane with inav5 and when I am in acro (stabilized) mode the rudder throw is very low.  I would like to use this plane for some aerobatics and I was wonder how I can increase the throw to come closer to what I can get in manual mode.

**Comment by DzikuVx** [comment, marked as answer]
You have to increase Yaw FF gains in Configurator. That's all

**Reply by NiteRavenX** [reply]
Actually FF didn't seem to change anything but increasing yaw rate to 100 deg/s did help, I was using plane defaults which I think was around 30.  I haven't flown it yet but on the bench the rudder movement has increased to closer to manual throw.
